src/content/apis/foods/FoodDatabaseAPI.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Edamam
+description: Get free access to a database with close to 900,000 foods and over 680,000 unique UPC codes.
+categories:
+  - foods
+features:
+  https: true
+url: https://developer.edamam.com/food-database-api
+logo: ![small-logo](https://user-images.githubusercontent.com/67019423/193456793-834cee23-1d15-4cfb-8eed-04bdaf733bdb.png)
```

Comment:
The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.